Gemfile

```diff
@@ -24,11 +24,13 @@ gem 'rails_autolink'
 # Please don't update hoptoad_notifier to airbrake.
 # It's for internal use only, and we monkeypatch certain methods
 gem 'hoptoad_notifier', "~> 2.4"
-gem 'draper', :require => false
+gem 'draper'
 
 gem 'errbit_plugin', github: 'errbit/errbit_plugin'
 gem 'errbit_github_plugin', github: 'errbit/errbit_github_plugin'
 
+gem 'dotenv-deployment'
```

Comment:
1) errbit is already using two configuration paradigms - config.yml and ENV (e.g. config/initializers/mongo.rb). Therefore adding dotenv-deployment does not make big changes in configuration methods in errbit - it justs add easy way to configure it when deploying.

2) dotenv-deployment is pretty minimalistic as you described - just load .env to ENV. It does not do any other magic. For example, if ENV is manually passed, values from .env will not overwrite it, so there is (and should be) no harmful side effect.

3) Also note that secret.yml has default value for secret_key_base, so you do not need to create .env file for development.

Without dotenv-deployment, I cannot use capistrano "out-of-box" since I'm not deploying to Heroku. That's the main reason I think it should be part of this pull request. Probably I should have made separate pull request prior to this one, but all things happened at the same time on my side - sorry about that :)

So my proposal is 1) merge this request with dotenv-deployment and then 2) simplify configuration in some way.